client/src/containers/signup/signup.js

```diff
@@ -220,6 +222,7 @@ class Signup extends Component {
       <div className="signupContainer">
         <h1 className="center">Sign up</h1>
         <input
+          className='input'
```

Comment:
Why are you adding classname inputs here?

Reply:
I don't think it's necessary

Reply:
Fixed it :ok_hand: